src/NUnitEngine/nunit.engine.core/nunit.engine.core.csproj

```diff
@@ -9,6 +9,13 @@
     <Reference Include="System.Runtime.Remoting" />
     <Reference Include="System.Web" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0' or '$(TargetFramework)'=='netcoreapp3.1'">
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.3.0" />
+    <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <PackageReference Include="System.Reflection" Version="4.3.0" />
+    <PackageReference Include="System.Diagnostics.Process" Version="4.3.0" />
+    <PackageReference Include="System.Xml.XPath.XmlDocument" Version="4.3.0" />
+  </ItemGroup>
```

Comment:
Is this a rebase issue? Think these are the ones I removed just recently. 🙂

Reply:
I wrote and modified this as bugs arose around preview and other pre-release installations. I added a comment.

Reply:
@CharliePoole - I think this one's been missed too. If I'm understanding right, your comment above refers to the logic in RuntimeFramework.cs. (And is very helpful, thank you! 😆)

My original comment on this file still stands.

Reply:
It turns out that only the Microsoft.Win32.Registry reference was needed.